src/OceanSimulations/OceanSimulations.jl

```diff
@@ -41,5 +41,6 @@ default_or_override(override, alternative_default=nothing) = override
 
 include("barotropic_potential_forcing.jl")
 include("ocean_simulation.jl")
+include("prescribed_ocean_model.jl")
```

Comment:
is this used in the arctic simulation?

Reply:
no, I can separate it out, it is there because I started with a prescribed ocean, then I fell back to a prognostic ocean. I'll leave this to another PR